### PR TITLE
Security fixes

### PR DESCRIPTION
**Pull request by Vilican**
I have fixed 2 non-critical security vulnerabilities:

 * login forms don't have CSRF tokens
 * links using target="_blank" in footer don't use rel="noopener"

### CODE REVIEW

File: login.php [hunk 1]
```diff
@@ -23,10 +23,11 @@
  *
  *  fUsername
  *  fPassword
+ *  token
  *  lang
  */
 
-define('POSTFIXADMIN_LOGOUT', 1);
+//define('POSTFIXADMIN_LOGOUT', 1);
```
Comment (cboltz) [inline login.php]: Most of your changes make sense and are a good idea, but - why do you drop the POSTFIXADMIN_LOGOUT define? It is used in common.php to log out a user whenever going to login.php, and without that define that won't happen (and you are still logged in)...

Comment (Vilican) [inline login.php]: If you look carefuly on the code in common.php, you will see this:

```
if (defined('POSTFIXADMIN_LOGOUT')) {
    session_unset();
    session_destroy();
    session_start();
}
```

My code stores CSRF token in `$_SESSION` and this code deletes the session (including the token).

If you actually wanted to destroy session (log out) of the old user whenever new user logs in, you would use `session_regenerate_id (true)` - the true means destroy old session.

Comment (cboltz) [inline login.php]: Feel free to adjust the logout code with your proposal ;-)

I just tested your change and always get "invalid token" when trying to login :-( Can you please check what is wrong?

**Comment by Vilican** [comment]
I have added back the logout mechanism. Also, I forgot to commint the change in template with CSRF token - fixed now.

**Pull request merged**